Bindings/Python/simple-arm.py

```diff
@@ -0,0 +1,132 @@
+# You may obtain a copy of the License at                                 %
+# http://www.apache.org/licenses/LICENSE-2.0.                             %
+#                                                                         % 
+# Unless required by applicable law or agreed to in writing, software     %
+# distributed under the License is distributed on an "AS IS" BASIS,       %
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+# implied. See the License for the specific language governing            %
+# permissions and limitations under the License.                          %
+# ----------------------------------------------------------------------- %
+
+# simple-arm.py                                                        
+# Author: Neil Dhir
+
+import opensim as osim
+
+# Define global model where the arm lives.
+arm = osim.Model()
+
+# ---------------------------------------------------------------------------
+# Create two links, each with a mass of 1 kg, centre of mass at the body's
+# origin, and moments and products of inertia of zero.
+# ---------------------------------------------------------------------------
+
+# chrisdembia: use 4 spaces instead of tabs
+humerus = osim.Body('humerus',
+	1.0,
+	osim.Vec3(0, 0, 0),
+	osim.Inertia(0, 0, 0))
+radius = osim.Body('radius',
+	1.0,
+	osim.Vec3(0, 0, 0),
+	osim.Inertia(0, 0, 0))
+
+# ---------------------------------------------------------------------------
+# Connect the bodies with pin joints. Assume each body is 1m long.
+# ---------------------------------------------------------------------------
+
+shoulder = osim.PinJoint("shoulder",
+	arm.getGround(), 	    		# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 0, 0),
+	humerus, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 1, 0))
+
+elbow = osim.PinJoint("elbow",
+	humerus, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 0, 0),
+	radius, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 1, 0))
+
+# ---------------------------------------------------------------------------
+# Add a muscle that flexes the elbow (actuator for robotics people).
+# ---------------------------------------------------------------------------
+
+biceps = osim.Millard2012AccelerationMuscle("biceps", 	# 	Muscle name
+   	200.0, 	 	#	Max isometric force
+    	0.6, 		# 	Optimal fibre length
+    	0.55, 		# 	Tendon slack length
+    	0.0)		# 	Pennation angle
+biceps.addNewPathPoint("origin",						
+   	humerus, 						
+   	osim.Vec3(0, 0.8, 0))			
+
+biceps.addNewPathPoint("insertion",
+   	radius,
+   	osim.Vec3(0, 0.7, 0))
+
+# # ---------------------------------------------------------------------------
+# # Add a controller that specifies the excitation of the muscle.
+# # ---------------------------------------------------------------------------
+
+brain = osim.PrescribedController()
+brain.addActuator(biceps)
+brain.prescribeControlForActuator(1,					
```

Comment:
Thank you for the additional information. I created #1178 to fix the bug that you found. Once #1178 is merged, it will work to pass in the string name of the actuator. It is definitely incorrect to pass `1` as the index; the error message was just not making its way up to python.

For this PR, just use the string name instead. The example will not work until #1178 is also merged, but at least then you won't have to wait for #1178 to get merged for this PR to be merged.